scripts/lib/mkcloud-common.sh

```diff
@@ -117,7 +117,7 @@ function safely
 
 function rubyjsonparse
 {
-    $ruby -e "
+    ruby -e "
```

Comment:
loosk like that `ruby` is defined in qa_crowbarsetup.sh to point to 2.1. Is there a reason to skip this here?